client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Box, Typography, Input, InputLabel, Button, IconButton } from '@mui/material';
+import Avatar from '@mui/material/Avatar';
+import SettingHeader from '../SettingsHeader/SettingsHeader';
+import DeleteIcon from '@mui/icons-material/Delete';
+
+const ProfilePhoto = (): JSX.Element => {
+  const [file, setFile] = useState<File>();
+
+  const handleImgChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (!e.target.files) return;
+    setFile(e.target.files[0]);
+  };
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+      <SettingHeader header="Profile Photo" />
+      <Avatar
+        alt="profile photo"
+        src={`https://www.nj.com/resizer/zovGSasCaR41h_yUGYHXbVTQW2A=/1280x0/smart/cloudfront-us-east-1.images.arcpublishing.com/advancelocal/SJGKVE5UNVESVCW7BBOHKQCZVE.jpg`}
+        sx={{ width: 120, height: 120, marginBottom: 4 }}
+      />
+      <Box textAlign="center">
+        <Typography sx={{ fontSize: 15, fontWeight: 600, color: '#afafaf' }}>
```

Comment:
Let's consider using a `Typography` variant for font size (we can set custom font sizes for variants in MUI's theme) and also use the theme to apply font color

Ditto where applicable